internal/core/workers/operation_execution_worker/metrics.go

```diff
@@ -0,0 +1,90 @@
+package operation_execution_worker
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/topfreegames/maestro/internal/core/monitoring"
+)
+
+const (
+	LabelNoOperationExecutorFound = "no_operation_executor_found"
+	LabelShouldNotExecute         = "should_not_execute"
+	LabelNextOperationFailed      = "next_operation_failed"
+	LabelStartOperationFailed     = "start_operation_failed"
+)
+
+var (
+	operationExecutionMetricOpts = monitoring.MetricOpts{
+		Namespace: monitoring.Namespace,
+		Subsystem: monitoring.SubsystemWorker,
+		Name:      "operation_execution",
+		Help:      "An scheduler operation was executed",
+		Labels: []string{
+			monitoring.LabelScheduler,
+			monitoring.LabelOperation,
+			monitoring.LabelSuccess,
+		},
+	}
+	operationExecutionLatencyMetric = monitoring.CreateLatencyMetric(&operationExecutionMetricOpts)
+
+	operationOnErrorMetricOpts = monitoring.MetricOpts{
+		Namespace: monitoring.Namespace,
+		Subsystem: monitoring.SubsystemWorker,
+		Name:      "operation_on_error",
+		Help:      "An scheduler operation on error fallback was executed",
+		Labels: []string{
+			monitoring.LabelScheduler,
+			monitoring.LabelOperation,
+			monitoring.LabelSuccess,
+		},
+	}
+	operationOnErrorLatencyMetric = monitoring.CreateLatencyMetric(&operationOnErrorMetricOpts)
+
+	operationEvictedCountMetricOpts = monitoring.MetricOpts{
+		Namespace: monitoring.Namespace,
+		Subsystem: monitoring.SubsystemWorker,
+		Name:      "operation_evicted",
+		Help:      "An scheduler operation was evicted",
+		Labels: []string{
+			monitoring.LabelScheduler,
+			monitoring.LabelOperation,
+			monitoring.LabelReason,
+		},
+	}
+	operationEvictedCountMetric = monitoring.CreateCounterMetric(&operationEvictedCountMetricOpts)
+
+	operationExecutionWorkerFailedCountMetricOpts = monitoring.MetricOpts{
+		Namespace: monitoring.Namespace,
+		Subsystem: monitoring.SubsystemWorker,
+		Name:      "operation_execution_worker_failed",
+		Help:      "An scheduler operation execution worker failed and is no longer running",
+		Labels: []string{
+			monitoring.LabelScheduler,
+			monitoring.LabelReason,
+		},
+	}
+	operationExecutionWorkerFailedCountMetric = monitoring.CreateCounterMetric(&operationExecutionWorkerFailedCountMetricOpts)
+)
+
+func ReportOperationExecutionLatency(start time.Time, schedulerName, operationName string, success bool) {
```

Comment:
nit: We could keep those functions private since no other package will use them.